auto/src/glew_tail.h

```diff
@@ -8,10 +8,10 @@
 #define GLEW_ERROR_GLX_VERSION_11_ONLY 3  /* Need at least GLX 1.2 */
 
 /* string codes */
-#define GLEW_VERSION 1
-#define GLEW_VERSION_MAJOR 2
-#define GLEW_VERSION_MINOR 3
-#define GLEW_VERSION_MICRO 4
+#define GLEW_VERSION GLEW_VERSION_MAJOR_STRINGGLEW_VERSION_MINOR_STRING
+#define GLEW_VERSION_MAJOR GLEW_VERSION_MAJOR_STRING
+#define GLEW_VERSION_MINOR GLEW_VERSION_MINOR_STRING
+#define GLEW_VERSION_MICRO GLEW_VERSION_MICRO_STRING
 
```

Comment:
I'll take another look at this myself eventually, unless a fresh merge request appears. :-)